creator-node/src/ffmpeg.js

```diff
@@ -3,13 +3,14 @@ const fs = require('fs')
 const path = require('path')
 const ffmpeg = require('ffmpeg-static').path
 const spawn = require('child_process').spawn
+const { logger } = require('./logging')
 
 /** Segments file into equal size chunks without re-encoding
  *  Try to segment as mp3 and error on failure
  */
-function segmentFile (req, fileDir, fileName) {
+function segmentFile (fileDir, fileName) {
```

Comment:
the reason we plumb req all the way through is there's a request id that get logged out so we can associate all the steps for a given request. we'd lose that here. is there any way to add that to the queue to preserve this?

Reply:
https://github.com/AudiusProject/audius-protocol/blob/master/creator-node/src/logging.js#L27

Reply:
AFAIK no because bull needs to serialize the job payload and it couldn't. I can try and and figure out another way to plumb it thru

Reply:
Ah ok. Solution is to pass the log context thru. Lmk what you think